lib/sycamore/sycamore/docset.py

```diff
@@ -974,29 +980,51 @@ def llm_filter(
             new_field: The field that will be added to the DocSet with the outputs.
             prompt: LLM prompt.
             field: Document field to filter based on.
-            threshold: Cutoff that determines whether or not to keep document.
+            threshold: Cutoff that determines whether to keep document.
+            keep_none:  whether to keep records with a None value.
+                        Warning: using this might hide data corruption issues.
+            use_elements: use contents of a document's elements to filter as opposed to document level contents.
+            similarity_query: query string to compute similarity against.
+            similarity_scorer: scorer to generate similarity if 'sort_elements_by_similarity' is True.
```

Comment:
What is `sort_elements_by_similarity` and where is it set?

Reply:
Ah leftover comment